### DISCUSSION TITLE
Announcing .NET 8 support for Static Web Apps

### DISCUSSION THREAD

**Discussion by thomasgauvin** [Azure/static-web-apps#1347] [Announcements]
Last week, .NET 8 was announced at Microsoft Ignite, with .NET 8 support coming to Azure Static Web Apps! Now, you can build .NET 8 Blazor WebAssembly frontend web applications. You can also complement your static sites with .NET 8 managed Azure Functions, in order to add backend functionality to your frontend sites.

## Get started with .NET 8 on Azure Static Web Apps
See our latest update here: [Generally available: Azure Static Web Apps now supports .NET 8](https://azure.microsoft.com/en-ca/updates/generally-available-azure-static-web-apps-now-supports-net-8/)
Follow our Azure Static Web Apps Blazor tutorial: Learn more about [Azure Static Web Apps Blazor Support](https://aka.ms/swa-blazor-tutorial).

Azure Static Web Apps support for .NET 8 follows the .NET 8 lifecycle. To learn more, please refer to the .[NET support policy.](https://dotnet.microsoft.com/platform/support/policy)

**Comment by thomasgauvin** [comment]
We published a blog to show you how to get started, with helpful resources such as a template repository for a full-stack .NET 8 and Blazor application deployed to Azure Static Web Apps. Check it out: https://techcommunity.microsoft.com/t5/apps-on-azure-blog/build-and-deploy-net-8-blazor-wasm-apps-with-serverless-apis/ba-p/3988412

**Comment by eolamisan** [comment]
I'm getting the following error when trying to deploy


staticwebapp.config.json

```
    "platform": {
      "apiRuntime": "dotnet-isolated:8.0"
    }
```

Error

```
Welcome to Azure Static Web Apps CLI (1.1.6)

...

The following error was encountered: must be equal to one of the allowed values dotnet:3.1, dotnet:6.0, dotnet-isolated:6.0, dotnet-isolated:7.0, node:12, node:14, node:16, node:18, python:3.8, python:3.9, python:3.10
7:     },
8:     "platform": {
9:         "apiRuntime": "dotnet-isolated:8.0"
10:     }

```

**Reply by ciacco85** [reply]
me too...

**Comment by danbasszeti** [comment]
I've been carrying out the upgrade and I'm concerned with a bit of a drop in cold start performance - will a Managed Function take advantage of ReadyToRun, and if so how do I do it? As far as I can tell the managed functions run on windows, but the GitHub action to deploy them only runs on Linux.
